src/wallet/rpcwallet.cpp

```diff
@@ -808,16 +810,19 @@ UniValue getbalancedetail(const UniValue& params, bool fHelp)
 
         for (auto const& s : listSent)
         {
+            std::string address;
+
             CBitcoinAddress addr;
             addr.Set(s.destination);
+            address = addr.Set(s.destination) ? addr.ToString() : std::string {};
```

Comment:
maybe `address` should be something like `addressStr` to help signify what's different about it from `addr`

Reply:
Isn't that obvious? A few lines above address is declared with std::string, addr has type CBitcoinAddress.

Reply:
Fine for now. All the `CBitcoinAddress` stuff is changing with the Bitcoin rebase anyway, so this won't last forever.

Reply:
It is if you are reading it, but for skimming it makes you to go back and reread since you might not have looked at the type closely. Not a huge deal, but just a minor thing